test/sqllogictest/chbench.slt

```diff
@@ -991,36 +1013,45 @@ AND total_revenue = (
 ORDER BY su_suppkey
 ----
 %0 =
+| Get materialize.public.supplier (u34)
+
+%1 =
 | Get materialize.public.orderline (u19)
 | ArrangeBy (#5, #4)
 
-%1 =
+%2 =
 | Get materialize.public.stock (u26)
 | ArrangeBy (#0, #1)
 
-%2 =
-| Join %0 %1 (= #5 #11) (= #4 #10)
-| | implementation = DeltaQuery %0 %1.(#0, #1) | %1 %0.(#5, #4)
+%3 =
+| Join %1 %2 (= #5 #11) (= #4 #10)
+| | implementation = DeltaQuery %1 %2.(#0, #1) | %2 %1.(#5, #4)
 | | demand = (#6, #8, #27)
 | Filter (datetots(#6) >= 2007-01-02 00:00:00)
 | Reduce group=(#27) sum(#8)
-
-%3 =
-| Get materialize.public.supplier (u34)
+| ArrangeBy (#0)
 
 %4 =
-| Get %2
-| ArrangeBy (#0)
+| Get materialize.public.orderline (u19)
+| ArrangeBy (#5, #4)
 
 %5 =
-| Get %2
+| Get materialize.public.stock (u26)
+| ArrangeBy (#0, #1)
+
+%6 =
+| Join %4 %5 (= #5 #11) (= #4 #10)
+| | implementation = DeltaQuery %4 %5.(#0, #1) | %5 %4.(#5, #4)
+| | demand = (#6, #8, #27)
+| Filter (datetots(#6) >= 2007-01-02 00:00:00)
+| Reduce group=(#27) sum(#8)
```

Comment:
This looks like a legit opportunity for deduplication. It will get deduplicated in rendering, but we could in principle notice that here and produce a cleaner plan.